products/jbrowse-desktop/package.json

```diff
@@ -92,23 +93,26 @@
   },
   "homepage": "./",
   "build": {
-    "electronVersion": "7.1.0",
+    "electronVersion": "9.1.1",
```

Comment:
Maybe we should pin this to a specific version in package.json so it doesn't get out of sync

Reply:
went ahead and did this, but in general I don't like pinning versions. it seemed like the ideal is it gets electron version from package.json but couldn't get this to work